### PR TITLE
Add support for extending specific classes

### PR DESCRIPTION
**Pull request by Lucianovici**
Enhance `SCHEDULER_BASE_CLASSES` setting.

### CODE REVIEW

File: schedule/utils.py [hunk 1]
```diff
@@ -233,10 +232,16 @@ def coerce_date_dict(date_dict):
     return modified and ret_val or {}
 
 
-def get_model_bases():
+def get_model_bases(model_class_name):
```
Comment (llazzaro) [inline schedule/utils.py]: :+1:

File: tests/test_utils.py [hunk 2]
```diff
@@ -130,3 +132,41 @@ def test_get_occurrence_works_for_event_like_object(self):
             expected_ok = True
 
         assert expected_ok
+
+
+class BaseTestClass(object):
+    pass
```
Comment (llazzaro) [inline tests/test_utils.py]: Please delete this empty test class

Comment (Lucianovici) [inline tests/test_utils.py]: Agree, it's ugly, but how to test the `import_string`. What I've written here is not really a unit test.
I should have use `mock` lib to mock that `import_string`. Should I add `mock`? I see you're not using it.

**Comment by Lucianovici** [comment]
@llazzaro I updated, can you please take a look to the other CI errors? I can't figure it out right now. Thanks.

**Comment by llazzaro** [comment]
@jwicks thanks. can you give the the hashes of those commits?

**Comment by jwicks** [comment]
2fa2d358c4ffcedbe41572e1180bfb61091247f9 from https://github.com/jwicks/django-scheduler/
59e71f13cbaa1830b2fda2504345be068756d98c from https://github.com/jwicks/django-scheduler/

Let me know if you'd prefer if I re-PR my changes to llazzaro:develop

**Comment by llazzaro** [comment]
this PR has been merged on commits:
https://github.com/llazzaro/django-scheduler/commit/77ead26684d9dcea61ae5038aa38e159906bc826
https://github.com/llazzaro/django-scheduler/commit/35f0b8cc2476a2455624beaeba2564a236f63805
https://github.com/llazzaro/django-scheduler/commit/7861232b50aaad7ab6eef097d01a7c6af13cf9e7
https://github.com/llazzaro/django-scheduler/commit/387fd66badfbe2bc021f476c4c26f2d1b4aab6d8

**Comment by Lucianovici** [comment]
Thanks guys! :)

**Pull request closed without merging**